frontend/prebuild/src/app/login/login.component.ts

```diff
@@ -34,13 +34,14 @@ export class LoginComponent implements OnInit, OnDestroy {
   party: string;
   queuePosition: number;
   player = new Player();
+
   isFullDevice: boolean = !/Android|webOS|iPhone|iPad|iPod|BlackBerry|IEMobile|Opera Mini/i.test(navigator.userAgent);
   isQuickPlayAllowed: boolean = this.isFullDevice;
-  isSingleParty: boolean = false;
```

Comment:
why do we want to remove the type from these variables?

Reply:
"Type trivially inferred from boolean literal"